includes/core-changes/codeanalysis/5.0/outattributes-on-pinvoke-string-parameters.md

```diff
@@ -0,0 +1,56 @@
+### CA1417: OutAttribute on string parameter for P/Invoke
+
+.NET code analyzer rule [CA1417](/visualstudio/code-quality/ca1417) is enabled, by default, starting in .NET 5.0. It produces a build warning for any [Platform Invoke (P/Invoke)](../../../../docs/standard/native-interop/pinvoke.md) method definitions where a <xref:System.String> parameter is marked with <xref:System.Runtime.InteropServices.OutAttribute>.
```

Comment:
It is only where a `string` parameters is marked with `OutAttribute` and passed by value. I think the 'passed by value' is worth including (the CA1417 doc includes it), since the change description below mentions passing an interned string by value (and recommended action mentions changing to passing by reference).